packages/graphql-codegen-cli/src/config.ts

```diff
@@ -74,15 +74,16 @@ function customLoader(ext: 'json' | 'yaml' | 'js' | 'ts') {
     }
 
     if (ext === 'ts') {
-      // #8437: conflict with `graphql-config` also using TypeScriptLoader(), causing a double `ts-node` register.
-      const tsLoader = TypeScriptLoader({ transpileOnly: true });
-      return tsLoader(filepath, content);
+      const jiti = require('jiti')(__filename, {
```

Comment:
You can use `import`

Reply:
Ah right. We do bundle to both. What's the best way to deal with it? Something like this: 
```
import { fileURLToPath } from 'node:url';
const __filename = fileURLToPath(import.meta.url)
```
?

Reply:
@enisdenjo could we add banner with bob? Or should we bundle to Esm only instead?

Reply:
@beerose yeah but we need to add it only to ESM build 😅

Reply:
@B2o5T this is what was generated in `graphql-config` in ESM out of the box:

```
const __filename = fileURLToPath(import.meta.url);
```

What's the difference between the build setup? Ie can we have this also in this package?

Reply:
@beerose i added it manually in your PR 😅 https://github.com/kamilkisiela/graphql-config/blob/master/scripts/postbuild.ts

Reply:
should we do the same thing here?

Reply:
@saihaj, do you have any thoughts on that?

Reply:
I think our script is fine for simplicity. We do some binary fixing too with scripts. Ideally `bob` should have some `hooks` that we can use to call scripts as needed.

Reply:
> should we do the same thing here?

if there is no other workaround..

Reply:
Ok, so it looks like we can workaround it but not provide __filename at all — jiti resolves it to `process.cwd()` which I guess it's correct? https://github.com/unjs/jiti/blob/main/src/jiti.ts#L101